openssh/README.md

```diff
@@ -0,0 +1,132 @@
+# openssh
+
+## Maintainers
+
+The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+## Testing
+
+### Build the openssh core plan
+
+Enter a habitat studio
+
+```bash
+hab pkg build .
+```
+
+Exit habitat studio
+
+### Build and install the test plan
+
+Enter another habitat studio
+
+```bash
+set -x
+
+# set environment from the last openssh build
+source results/last_build.env
+
+# stop the habitat supervisor
+hab sup term
+
+# add some required software before building the openssh_test
+hab pkg install -b core/hab-studio
```

Comment:
Believe we don't need this package install.
I suggest doing separate investigation if this is needed under certain conditions.